src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -261,8 +258,8 @@ public StudentExamResource(ExamAccessService examAccessService, StudentExamServi
         }
         var studentExam = optionalStudentExam.get();
 
-        // check that the studentExam is over, otherwise /studentExams/conduction should be used
-        if (!studentExam.isSubmitted() && !studentExam.isEnded()) {
+        // check that the studentExam is over and that it has been submitted, otherwise /studentExams/conduction should be used
+        if (!studentExam.isSubmitted() || !studentExam.isEnded()) {
```

Comment:
I think this still isn't right. Scenario hand-in early: isSubmitted = true & isEnded = false. This would lead to forbidden(). Couldn't we just use isSubmitted? If the student handed in his exam then his exam is definitely over.

Reply:
See 8519b5